src/vizy/gclouddialog.py

```diff
@@ -161,6 +163,20 @@ def func():
                 result += self.error_text.out_value(f"An error occurred: {e}") + self.error_dialog.out_open(True)
             return result
 
+        @self.test_sheet.callback()
+        def func():
+            self.kapp.push_mods(self.test_sheet.out_spinner_disp(True), callback_context.client)                        
+            gpsm = self.gcloud.get_interface("KtabularClient")
+            result = self.test_sheet.out_spinner_disp(False)
+            try:
+                data = util.testing.makeDataFrame() # creates a data frame with random values for testing
```

Comment:
IMO a good test sheet would have the date/time in it somehow... text is ok too so you can make it more readable.

Reply:
Did you catch my comment about making changing the sheet?

This is a test sheet
Created on 06 24 2022 19:57:23  

Maybe put each word in a separate column.  This is good because it verifies that things are working well.  Random data is just random...